tests/testthat/test-fct_relabel.R

```diff
@@ -36,3 +36,10 @@ test_that("additional arguments", {
 
   expect_identical(fct_relabel(f1, paste0, "."), factor(paste0(letters, ".")))
 })
+
+test_that("function coercion", {
+  f1 <- factor(letters)
+
+  expect_identical(fct_relabel(f1, ~paste0(.x, ".")),
```

Comment:
Can you please use the formatting described at http://style.tidyverse.org/syntax.html#long-lines ?